planck-cljs/test/planck/socket/alpha_test.cljs

```diff
@@ -28,7 +28,7 @@
 (defn inc! [r]
   (swap! r inc))
 
-(deftest listen-protected-port
+#_(deftest listen-protected-port
```

Comment:
The ARM CI box evidently doesn't balk if you bind on a protected port.

Reply:
macOS lets you do it, too, right? Hence the conditional on the next line?

Reply:
Ahh, right.